File: src/commands/userinfo.js
module.exports = {
    name: 'userinfo',
    description: 'Sends the user\'s public information',
    execute(client, message, args) {
        const user = message.author;
        const embed = {
            title: `${user.username}'s Public Information`,
            fields: [
                { name: 'Username', value: user.username, inline: true },
                { name: 'Discriminator', value: user.discriminator, inline: true },
                { name: 'ID', value: user.id, inline: true },
                { name: 'Status', value: user.presence.status, inline: true },
                { name: 'Created At', value: user.createdAt, inline: true }
            ],
            timestamp: new Date(),
            footer: {
                text: `Requested by ${user.username}`,
                icon_url: user.avatarURL()
            },
        };
        message.channel.send({ embed });
    },
};

File: scripts/commands/userinfo.py
import discord
from redbot.core import commands

class UserInfo(commands.Cog):
    """Sends information about the user who sent the -userinfo command"""

    @commands.command()
    async def userinfo(self, ctx):
        """Sends information about the user who sent the command"""
        user = ctx.message.author
        embed = discord.Embed(title=f"User Information - {user}", color=0x00ff00)
        embed.add_field(name="Name", value=user.name, inline=True)
        embed.add_field(name="ID", value=user.id, inline=True)
        embed.add_field(name="Status", value=user.status, inline=True)
        embed.add_field(name="Highest Role", value=user.top_role, inline=True)
        embed.add_field(name="Created At", value=user.created_at, inline=True)
        await ctx.send(embed=embed)

def setup(bot):
    bot.add_cog(UserInfo())
